website/versioned_docs/version-0.61/native-modules-setup.md

```diff
@@ -1,232 +1,311 @@
+
+### Adding React Native Windows to the Visual Studio Solution
+
+We're going to add several React Native Windows projects to your solution. So to avoid confusing them with your own code, we're first going to create a solution folder called `ReactNative`:
+
+1. Open the Solution Explorer sidebar.
+1. Right-click on `Solution 'MyLibrary'` at the top.
+1. Select `Add` > `New Solution Folder`.
+1. Name the folder `ReactNative`.
+
+Now we're going to add all of the following React Native Windows projects to that `ReactNative` folder. All of these projects are located under the `node_modules\react-native-windows` directory in the root of your `react-native-my-library` project directory.
+
+>*For more details about what these projects do, see [Project Structure](https://github.com/microsoft/react-native-windows/blob/0.61-stable/docs/project-structure.md).*
+
+| VS Project                          | Project File                                                                     |
+| :---------------------------------- | :------------------------------------------------------------------------------- |
+| Common                              | `Common\Common.vcxproj`                                                          |
+| Folly                               | `Folly\Folly.vcxproj`                                                            |
+| JSI.Universal                       | `JSI\Universal\JSI.Universal.vcxproj`                                            |
+| Microsoft.ReactNative               | `Microsoft.ReactNative\Microsoft.ReactNative.vcxproj`                            |
+| ReactCommon                         | `ReactCommon\ReactCommon.vcxproj`                                                |
+| ReactWindowsCore                    | `ReactWindowsCore\ReactWindowsCore.vcxproj`                                      |
+| Chakra                              | `Chakra\Chakra.vcxitems`                                                         |
+| JSI.Shared                          | `JSI\Shared\JSI.Shared.vcxitems`                                                 |
+| Microsoft.ReactNative.Cxx           | `Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems`                   |
+| Microsoft.ReactNative.SharedManaged | `Microsoft.ReactNative.SharedManaged\Microsoft.ReactNative.SharedManaged.shproj` |
+| Mso                                 | `Mso\Mso.vcxitems`                                                               |
+| Shared                              | `Shared\Shared.vcxitems`                                                         |
+
+For each project, you'll do the following:
+
+1. Open the Solution Explorer sidebar.
+1. Right-click on the `ReactNative` folder.
+1. Select `Add` > `Existing Project...`.
+1. Select the project file and click `Open`.
+
+When you are done, your solution should look like this:
+![native module dependencies](assets/native-module-dependencies61.png)

```

Comment:
I wonder if there's a way we could log recurring work for bits like this that need to be updated for each release. I get the feeling we could otherwise just forget. Part of me thinks it might be better to leave out bits that we think would get quickly outdated.

Reply:
I think we need explicit owners of each doc that are responsible for keeping them up to date.